vochain/offchaindatahandler/offchaindatahandler.go

```diff
@@ -26,6 +28,7 @@ type importItem struct {
 	itemType   int
 	uri        string
 	censusRoot string
+	pid        types.HexBytes
```

Comment:
i added this so offchaindatahandler can keep track of which process metadata it is, and tell indexer to update that process